components/sensor/bme680_bsec.rst

```diff
@@ -174,59 +178,100 @@ For each sensor all other options from :ref:`Sensor <config-sensor>` and :ref:`T
         # Default: static
         iaq_mode: static
 
-        # Sample rate
-        # -----------
-        # Available options:
-        # - lp (low power - samples every 3 seconds)
-        # - ulp (ultra low power - samples every 5 minutes)
-        # Default: lp
-        sample_rate: lp
-
         # Interval at which to save BSEC state
         # ------------------------------------
         # Default: 6h
         state_save_interval: 6h
 
     sensor:
       - platform: bme680_bsec
-        temperature:
+        temperature:        
```

Comment:
I can't see anything else wrong so I'll just point out these couple of bits of trailing whitespace!